ci/sandbox/build-pipeline.yaml

```diff
@@ -502,6 +512,55 @@ spec:
             trigger: true
           - get: tests-image
             passed: [build-proxy-node]
+          - get: eidas-config
+            trigger: true
+
+      - task: generate-config
+        config:
+          platform: linux
+          image_resource: *openjdk_image
+          inputs:
+            - name: eidas-config
+          outputs:
+            - name: eidas-config
+          run:
+            path: /bin/sh
+            args:
+              - -euc
+              - |
+                cd eidas-config/tools
+                ./gradlew run --args="generate-config --environment=test"
+
+      - task: create-config-map
+        config:
+          platform: linux
+          image_resource: *task_toolbox
+          inputs:
+            - name: eidas-config
+          outputs:
+            - name: manifests
+          params:
+            KUBERNETES_SERVICE_ACCOUNT: ((namespace-deployer))
+            KUBERNETES_TOKEN: ((namespace-deployer.token))
+            NAMESPACE: ((namespace-deployer.namespace))
+          run:
+            path: /bin/sh
+            args:
+              - -euc
+              - |
+                cd eidas-config/tools
+
+                echo "configuring kubectl"
+                echo "${KUBERNETES_SERVICE_ACCOUNT}" | jq -r .["ca.crt"] > ca.crt
+                kubectl config set-cluster self --server=https://kubernetes.default --certificate-authority=ca.crt
+                kubectl config set-credentials deployer --token "${KUBERNETES_TOKEN}"
+                kubectl config set-context deployer --user deployer --cluster self
+                kubectl config use-context deployer
+
+                echo "generating configMap resource definition from file"
+                kubectl -n "${NAMESPACE}" create configmap test-countries-config --from-file=countriesConfig.yaml -o yaml --dry-run \
```

Comment:
Maybe this should be a separate script as then it could be commented (I have no idea what the `num-versions` stuff is about.

Reply:
As in, a separate script in a file somewhere?

Reply:
Also, it's this: https://github.com/k14s/kapp/blob/master/docs/diff.md#versioned-resources

Reply:
Yeah, in a file was what I was thinking about. I understand now, sorta, what the version resources thing is, but I don't know how we'd end up deciding that 5 is the right magic number to keep :-)

Reply:
To be honest I just picked a sensible sounding number, and then later realised that 5 is actually the default. I left this is to be explicit. I think I'm gonna leave this as it is with regard to moving it to a new file/commenting. There's a link to the docs in the commit message if people are interested in what it's doing.